### ISSUE TITLE
Trunk column in Ratecards table is incorrect

### ISSUE THREAD

**Issue by JoseGoncalves** [radiusone/a2billing#9]
After I've imported a database from a running system (with A2Billing v2.3.0) I've noticed that the Trunk column in the Ratecards table is not correct, presenting always the same value, even when the data has different values.

Here goes a sample view of A2Billling v2.3.0 running with PHP 7.3:

![Screenshot from 2024-09-30 18-56-53](https://github.com/user-attachments/assets/d1ad4bc5-7237-4c8c-8e61-39708204eb47)

and the same data view with this new version running on PHP 8.2:

![Screenshot from 2024-09-30 18-57-16](https://github.com/user-attachments/assets/beff51fe-e110-45b4-874b-daa7a69b552e)

**Comment by miken32** [comment]
I suspect this is the same problem as with destination in the CDRs report, given the similar code for the column:
```php
$HD_Form -> AddViewElement(gettext("TRUNK"), "id_trunk", true, 15, "", "lie", "cc_trunk", "trunkcode", "id_trunk='%id'", "%1");
```
Aside from reordering some parameters, that line hasn't changed since 2.3. So likely the fix for both issues is somewhere else in the code, probably in the ViewHandler file. Will take a look.

**Comment by miken32** [comment]
Regarding this and the changes you made in #8, the reason those columns aren't part of the main query is because searches use the foreign table ID. So the query is done against the numeric value, and only when displaying the list the name is pulled from the foreign table. I don't know what "lie" means, but it is the view element type that has this special behaviour.

Anyway this should be fixed now with commit 83e51dd4; the caching of values to prevent repeated DB lookups was not working correctly. I also reverted those changes from #8 since the root cause was the same so CDRs should also be fixed.
